src/transformStateWithClones.js

```diff
@@ -7,7 +7,28 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateClone = { ...state };
```

Comment:
[NAMING] - use proper names for object copy. In this case, it would be better to name `stateClone` as `stateCopy`.